chamber-of-commerce.md

```diff
@@ -0,0 +1,93 @@
+# Woodcrest Chamber of Commerce
+## Business Approval Process
+*Effective Date: [Insert Date]*
+
+### 1. Eligibility
+1.1 Any individual who wishes to open a business in Woodcrest must be a member in good standing of the community.
+1.2 Businesses must be operated on designated commercial rental plots within the Woodcrest.
+1.3 Individuals applying for a business license must not have any criminal history recorded with the Woodcrest Police Department.
+1.4 Individuals must not have any outstanding fines or unresolved infractions with the Woodcrest Police Department at the time of application.
+
+### 2. Application Process
+2.1 Individuals must submit a business proposal to the Chamber of Commerce for review and approval before renting a commercial plot.
+2.2 The proposal must include:
+   a) Business name and type (e.g. retail, service, entertainment, etc.)
+   b) Detailed description of products/services offered
+   c) Target customer base and how it fits into the Woodcrest community
+   d) Proposed location (specific rental plot if known)
+   e) Any special requirements or requests (e.g. zoning variances)
+   f) Proof of any necessary permits or licenses (e.g. alcohol permit, health department certifications)
+2.3 Individuals are responsible for obtaining all required permits and licenses from the appropriate Woodcrest and North Carolina authorities before submitting their proposal.
+2.4 Proposals must be submitted via [designated in-game or out-of-game channel].
+
+### 3. Review Process
+3.1 The Chamber of Commerce will review each proposal to ensure it meets the following criteria:
+   a) Fits within the theme and setting of a university town in North Carolina
+   b) Does not significantly overlap with existing businesses in Woodcrest
+   c) Provides roleplay opportunities for the community
+   d) Complies with all sim rules and regulations
+3.2 The Chamber may request additional information or suggest changes to the proposal before making a decision.
+3.3 Proposals will be approved or denied within one week of submission.
+
+### 4. Post-Approval Steps
+4.1 Upon approval, individuals must complete the rental process for their designated commercial plot within one week.
```

Comment:
Given that this is the rental system, it's unlikely that this will ever be handed outside of administration.  I'll see what options we have with Dexy, but between the four of us, I would hope that a week to unlock a plot should be sufficient.

It really just comes down to what options are available from Dexy.